2020/hamming-codes/spanish/sentence_translations.json

```
@@ -443,7 +443,7 @@
  {
   "input": "So if you're the receiver, you look at this message, and you see an odd number of 1s, you can know for sure that some error has occurred, even though you might have no idea where it was.",
   "model": "nmt",
-  "translatedText": "Entonces, si usted es el receptor, mira este mensaje y ve un número impar de unos, puede estar seguro de que se ha producido algún error, aunque no tenga idea de dónde estaba.",
+  "translatedText": "Entonces, si tú eres el receptor, miras el mensaje y ves un número impar de unos, puedes estar seguro de que se ha producido algún error, aunque no tengas idea dónde estaba.",
```

I think the "de" in the last part of the sentence should stay there (I could not find a good reference, but I think the expression is "[tener idea de algo](https://www.collinsdictionary.com/es/diccionario/espanol-ingles/tener-idea-de-algo)"):

```suggestion
  "translatedText": "Entonces, si tú eres el receptor, miras el mensaje y ves un número impar de unos, puedes estar seguro de que se ha producido algún error, aunque no tengas idea de dónde estaba.",
```